doc/classes/CodeEdit.xml

```diff
@@ -153,7 +153,7 @@
 		<method name="do_indent">
 			<return type="void" />
 			<description>
-				Perform an indent as if the user activated the "ui_text_indent" action.
+				Perform an indent as if the user activated the [member ProjectSettings.input/ui_text_indent] action. If there is a selection this indents lines like [method indent_lines], otherwise indentation is added at the caret. Uses spaces or a tab depending on [member indent_use_spaces].
```

Comment:
```suggestion
				Perform an indent as if the user activated the [member ProjectSettings.input/ui_text_indent] action. If there is a selection, the selected lines are indented like [method indent_lines], otherwise indentation is added at the caret. Uses multiple spaces or a tab character depending on [member indent_use_spaces].
```
Or:
```suggestion
				Indents as if the user activated the [member ProjectSettings.input/ui_text_indent] action. If lines are selected, they are indented like [method indent_lines]. Otherwise, The additional indentation is added at the caret. The used characters depend on [member indent_use_spaces].
```
Or similar. Feel free to contest.